atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -48,27 +46,32 @@
     private final TableReference tableRef;
     private final OracleTableNameGetter oracleTableNameGetter;
     private final TableValueStyleCache valueStyleCache;
+    private OracleShrinkExecutor oracleShrinkExecutor;
 
     private OracleDdlTable(
             OracleDdlConfig config,
             ConnectionSupplier conns,
             TableReference tableRef,
             OracleTableNameGetter oracleTableNameGetter,
-            TableValueStyleCache valueStyleCache) {
+            TableValueStyleCache valueStyleCache,
+            OracleShrinkExecutor oracleShrinkExecutor) {
         this.config = config;
         this.conns = conns;
         this.tableRef = tableRef;
         this.oracleTableNameGetter = oracleTableNameGetter;
         this.valueStyleCache = valueStyleCache;
+        this.oracleShrinkExecutor = oracleShrinkExecutor;
     }
 
     public static OracleDdlTable create(
             TableReference tableRef,
             ConnectionSupplier conns,
             OracleDdlConfig config,
             OracleTableNameGetter oracleTableNameGetter,
-            TableValueStyleCache valueStyleCache) {
-        return new OracleDdlTable(config, conns, tableRef, oracleTableNameGetter, valueStyleCache);
+            TableValueStyleCache valueStyleCache,
+            OracleShrinkExecutor oracleShrinkExecutor) {
+        return new OracleDdlTable(config, conns, tableRef, oracleTableNameGetter, valueStyleCache,
+                oracleShrinkExecutor);
```

Comment:
How about shifting all parameters to this line, for consistency with other places in this PR?